addons/account/models/account_journal_dashboard.py

```diff
@@ -72,6 +72,26 @@ def _get_json_activity_data(self):
     show_on_dashboard = fields.Boolean(string='Show journal on dashboard', help="Whether this journal should be displayed on the dashboard or not", default=True)
     color = fields.Integer("Color Index", default=0)
     entries_count = fields.Integer(compute='_compute_entries_count')
+    has_holes = fields.Boolean(compute='_compute_has_holes')
+
+    def _query_has_holes(self):
+        self.env.cr.execute("""
+            SELECT move.journal_id,
+                   move.sequence_prefix
+              FROM account_move move
+             WHERE move.journal_id = ANY(%(journal_ids)s)
+               AND move.state = 'posted'
+          GROUP BY move.journal_id, move.sequence_prefix
+            HAVING COUNT(*) != MAX(move.sequence_number) - MIN(move.sequence_number) + 1
+        """, {
+            'journal_ids': self.ids,
```

Comment:
is it not leading to a traceback when `self` is an empty recordset?

Reply:
nope, because ` = ANY(...)`